addons/sale_timesheet/__init__.py

```diff
@@ -1,18 +1,6 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import api, SUPERUSER_ID
-
 from . import controllers
 from . import models
 from . import wizard
 from . import report
-
-def uninstall_hook(cr, registry):
-    env = api.Environment(cr, SUPERUSER_ID, {})
-    env['product.template'].search([
-        ('service_type', '=', 'timesheet')
-    ]).write({'service_type': 'manual'})
-    env['product.product'].search([
-        ('service_type', '=', 'timesheet')
-    ]).write({'service_type': 'manual'})
```

Comment:
todo: adapt selection field extension

Reply:
@rco-odoo this case cannot be handled by an ondelete='default' can it? It will set all `product.templates` to the default value since that's the model in which the field is defined but I doubt it applies to `product.product`, should we use a callback in this case?

Reply:
@Elkasitu actually the field `service_type` is defined on `product.template` and inherited in `product.product`.  Updating `product.template` records is enough, since *all* `product.product` records have a corresponding `product.template` record.  In fact, the second search/write above is totally useless.

Reply:
@rco-odoo Thanks for the clarification, in this case it is not even needed since the base field is not `required=True`, but I did it anyway to keep the behaviour consistent (by default, it would've been set to `NULL`